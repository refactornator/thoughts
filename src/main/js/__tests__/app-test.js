import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

jest.mock('../components/thought_input');
jest.mock('../components/thought_list');
import App from '../app';

describe('App', () => {
  it('renders the Thought Input and the Thought List', () => {
    const renderedApp = renderer
      .create(
        <MuiThemeProvider>
          <App />
        </MuiThemeProvider>
      )
      .toJSON();
    expect(renderedApp).toMatchSnapshot();
  });
});
