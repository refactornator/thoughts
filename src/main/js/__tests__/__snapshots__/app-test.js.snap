// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the Thought Input and the Thought List 1`] = `
<div
  className="container"
/>
`;
